app/models/gids_redis.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'common/models/redis_store'
+require 'common/models/concerns/cache_aside'
+
+# Facade for GIDS.
+class GIDSRedis < Common::RedisStore
+  include Common::CacheAside
+
+  REDIS_CONFIG_KEY = :gi_response
+  redis_config_key REDIS_CONFIG_KEY
```

Comment:
Since we don't need `REDIS_CONFIG_KEY` below, can be reduced to
`redis_config_key :gi_response`

Reply:
changed